src/Rasterization/Renderers/SVGTextRenderer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+namespace SVG\Rasterization\Renderers;
+
+use SVG\Rasterization\SVGRasterizer;
+
+class SVGTextRenderer extends SVGRenderer
+{
+    protected function prepareRenderParams(SVGRasterizer $rasterizer, array $options)
+    {
+        $options['x'] += $rasterizer->getOffsetX();
+        $options['y'] += $rasterizer->getOffsetY();
```

Comment:
Here, viewBox scaling and also output image scaling are not taken into account.

- x/y should be scaled in addition to the offset
- size should be scaled